### ISSUE TITLE
Docker image is huge (6.5 GB)

### ISSUE THREAD

**Issue by applejag** [srid/emanote#326]
**Describe the bug**
The Emanote Docker image is over 6 GB in size

```console
$ podman images sridca/emanote:latest
REPOSITORY                TAG         IMAGE ID      CREATED       SIZE
docker.io/sridca/emanote  latest      3fafc9417949  52 years ago  6.49 GB
```

(funny it says "CREATED 52 years ago" :hand_over_mouth:)

**To Reproduce**
1. `docker pull sridca/emanote:latest`
2. `docker images sridca/emanote:latest`
3. See results

**Expected behavior**
Docker image sizes are usually in the range of 1-200MB in size.

As the built `emanote` binary on my machine is ~200MB then the Docker image should be in that size range as well.

**Additional context**
For comparison of some images I already had locally:

```
// typical dev images:
REPOSITORY                    TAG         IMAGE ID      CREATED       SIZE
docker.io/library/php         latest      fbf532b99a6e  2 weeks ago   492 MB
docker.io/library/node        16          b9f398d30e45  5 weeks ago   931 MB
docker.io/library/golang      latest      738636f06010  2 months ago  988 MB

// extraordinary dev image:
REPOSITORY                      TAG                                   IMAGE ID      CREATED         SIZE
docker.io/unityci/editor        ubuntu-2021.3.7f1-linux-il2cpp-1.0.1  81a7c040c88b  2 days ago      7.39 GB

// typical app images:
REPOSITORY                    TAG         IMAGE ID      CREATED       SIZE
docker.io/library/nginx       alpine      f246e6f9d0b2  5 weeks ago   25 MB
quay.io/iver-wharf/wharf-web  latest      791531286f97  3 weeks ago   35.1 MB
docker.io/library/ubuntu      20.04       53df61775e88  3 months ago  75.1 MB

// extraordinary app image:
REPOSITORY                    TAG         IMAGE ID      CREATED       SIZE
docker.io/sridca/emanote      latest      3fafc9417949  52 years ago  6.49 GB
```

Almost as big as the Unity3D editor bundled with the IL2CPP compiler :P

Note: I'm using Podman, which lists the uncompressed file size. On Docker Hub it lists the compressed file size, which is only at 1GB (instead of 6.5 GB): https://hub.docker.com/layers/emanote/sridca/emanote/latest/images/sha256-c5d836ea139e51d4094baf0f4544aec2a123b53abd91b015a01ea3ce28acca5d?context=explore

**Comment by srid** [comment]
It used to be small. Possibly related: #295

**Comment by srid** [comment]
Resolved in 1.0.1.0 (#392)
